.gitignore

```diff
@@ -15,6 +15,10 @@ core/Enclave/*.o
 
 bin/
 
+ehsm-service/node_modules/
```

Comment:
change the folder name
ehsm-service -> ehsm-kms_service

Reply:
It will be fix.